Modules.md

```diff
@@ -11,6 +16,13 @@ is called a **module**. A module contains:
 Sections declare their type and byte-length. Sections with unknown types are
 silently ignored.
 
+An instance contains:
+* a [linear memory](AstSemantics.md#linear-memory);
+* [global variable](AstSemantics.md#global-variables) state;
+* (when [threading](PostMVP.md#threads) is added) thread-local state;
```

Comment:
That conversation isn't quite relevant: asm.js *already has TLS* in the form of plain old asm.js global variables (which are, by semantics of JS, thread-local).  In fact, the implementation of asm.js global access in Odin literally uses a pinned (TLS) register.  Jukka didn't realize this initially but I have since pointed it out to him so I think he's using them now.  Thus, it makes sense for Lars to say it doesn't make sense to put TLS *in SAB*; JS already has TLS in spades :)

However, in wasm, we're not defining our globals in terms of JS, so we don't get TLS "for free" like we do in asm.js; we have to define it (otherwise wasm will have a distinct regression compared to asm.js).  FWIW, I think TLS can be quite symmetric with shared globals such that global declarations can be any element of the product `{shared, thread-local}` x `{aliased, unaliased}`; I was talking about this with @dschuff the other day in the context of dynamic linking.